static/app/views/starfish/components/fullSpanDescription.tsx

```diff
@@ -26,14 +27,17 @@ export function FullQueryDescription({group, shortDescription}: Props) {
       <LoadingIndicator mini hideMessage relative />
     </PaddedSpinner>
   ) : (
-    <CodeSnippet language="sql">
-      {formatter.toString(description, {maxLineLength: LINE_LENGTH})}
+    <CodeSnippet language={snippetLanguage}>
+      {snippetLanguage === 'sql'
+        ? formatter.toString(description, {maxLineLength: LINE_LENGTH})
+        : description}
```

Comment:
l: The nested ternary is a little confusing, and it will require further refactoring when we add another language.

Let's just go with multiple returns.

```tsx
if (isLoading && isFetching) {
  return (
    <PaddedSpinner>
      <LoadingIndicator mini hideMessage relative />
    </PaddedSpinner>
  );
}

if (snippetLanguage === 'sql') {
  return (
    <CodeSnippet language={snippetLanguage}>
      formatter.toString(description, {maxLineLength: LINE_LENGTH})
    </CodeSnippet>
  );
}

return (
  <CodeSnippet>
    description
  </CodeSnippet>
);
```